Lib/asyncio/sslproto.py

```diff
@@ -196,11 +196,8 @@ def feed_ssldata(self, data, only_handshake=False):
 
             if self._state == _WRAPPED:
                 # Main state: read data from SSL until close_notify
-                while True:
-                    chunk = self._sslobj.read(self.max_size)
+                while (chunk := self._sslobj.read(self.max_size)):
```

Comment:
The new code calls `appdata.append(chunk)` one fewer time than the old code did.  It used to send the final "not chunk", but now it doesn't.

Reply:
Yeah this was previously noted in #8095

Reply:
The last (missing) `appdata.append(chunk)` can go into the `else:` clause of the `while`.